raft/src/main/java/org/apache/kafka/raft/ReplicatedLog.java

```diff
@@ -184,8 +184,10 @@ default ValidOffsetAndEpoch validateOffsetAndEpoch(long offset, int epoch) {
 
     /**
      * Flush the current log to disk.
+     *
+     * @param forceFlushActiveSegment Whether the flush includes the log end offset. Should be `true` during close; otherwise false.
```

Comment:
Whether the flush includes the log end offset => Whether to force flush the active segment?